src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -189,43 +190,48 @@ class ArchiveExplorer extends React.Component {
                 <div className="bp-ArchiveExplorer" data-resin-feature="archive">
                     <SearchBar onSearch={this.handleSearch} searchQuery={searchQuery} />
                     <Breadcrumbs fullPath={fullPath} onClick={this.handleBreadcrumbClick} view={view} />
-                    <VirtualizedTable
-                        rowData={itemList}
-                        rowGetter={this.getRowData(itemList)}
-                        sort={this.handleSort}
-                        sortBy={sortBy}
-                        sortDirection={sortDirection}
-                    >
-                        {intl => [
-                            <Column
-                                key={KEY_NAME}
-                                cellRenderer={itemNameCellRenderer(intl, this.handleItemClick)}
-                                dataKey={KEY_NAME}
-                                flexGrow={3}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('filename')}
-                                width={1}
-                            />,
-                            <Column
-                                key={KEY_MODIFIED_AT}
-                                cellRenderer={readableTimeCellRenderer}
-                                dataKey={KEY_MODIFIED_AT}
-                                flexGrow={2}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('last_modified_date')}
-                                width={1}
-                            />,
-                            <Column
-                                key={KEY_SIZE}
-                                cellRenderer={sizeCellRenderer()}
-                                dataKey={KEY_SIZE}
-                                flexGrow={1}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('size')}
-                                width={1}
-                            />,
-                        ]}
-                    </VirtualizedTable>
+                    <AutoSizer disableWidth>
+                        {({ height }) => (
+                            <VirtualizedTable
+                                height={height - 120} // Subtruct SearchBar and Breadcrumbs heights
```

Comment:
The need for this is counterintuitive, since the search bar and breadcrumbs are declared outside of the AutoSizer. Does the AutoSizer calculate its height based on the viewport? Does it still work if there are multiple previews on the same page?

Reply:
> The need for this is counterintuitive, since the search bar and breadcrumbs are declared outside of the AutoSizer. Does the AutoSizer calculate its height based on the viewport? Does it still work if there are multiple previews on the same page?

No it's not based on the viewport. AutoSizer expands to fill its parent but does not work well flex box. I added a block container and problem solved!